### PR TITLE
[Keyboard] Keycult 65

### PR DESCRIPTION
**Pull request by yiancar**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keycult/keycult65/keycult.c
Comment (waffle87) [inline keyboards/keycult/keycult65/keycult.c:1]: This file should be renamed to `keycult65.c`.

File: keyboards/keycult/keycult65/info.json
Comment (fauxpark) [inline keyboards/keycult/keycult65/info.json:1]: Please run `qmk format-json -i` on this file.

Comment (yiancar) [reply keyboards/keycult/keycult65/info.json:1]: done!

File: keyboards/keycult/keycult65/keycult.c
Comment (yiancar) [reply keyboards/keycult/keycult65/keycult.c:1]: phew good catch thank you!

**Review by waffle87 — APPROVED** [review]

**Review by keyboard-magpie — APPROVED** [review]

**Pull request merged**